### PR TITLE
Improved code styles

### PR DESCRIPTION
**Pull request by drakeet**
I am very pleased to see this project. However, I found some minor problems, and made corrections, hope that it will get better and better, to help more people find out the Agera.

The following are the updates:
- Changed `R.id.xxx_tv` to `R.id.xxx` and the field variable names
  associated with these IDs
- Fixed some hump naming problems such as `mValue1TVupdatable` to
  `mValue1Updatable`
- Reformated the XML layout file by
  [LayoutFormatter](https://github.com/drakeet/LayoutFormatter)
- Changed gradle version to `2.1.0` because the `2.2.0-alpha4` need
  Android Studio 2.2 preview 4

### CODE REVIEW

File: app/src/main/java/com/example/android/codelabs/agera/UiUtils.java [hunk 1]
```diff
@@ -21,18 +21,18 @@
 import android.view.animation.Animation;
 import android.view.animation.TranslateAnimation;
 
-public class UiUtils {
+public class UIUtils {
```
Comment (JoseAlcerreca) [inline app/src/main/java/com/example/android/codelabs/agera/UiUtils.java]: This is incorrect.

File: app/src/androidTest/java/com/example/android/codelabs/agera/step2/Step2ActivityFinalTest.java [hunk 2]
```diff
@@ -44,22 +44,22 @@
 
     @Test
     public void incrementButtonPressed_textIsUpdated() {
-        onView(withId(R.id.step2_value_tv)).check(matches(withText("0")));
-        onView(withId(R.id.increment_bt)).perform(click());
-        onView(withId(R.id.step2_value_tv)).check(matches(withText("1")));
+        onView(withId(R.id.step2_value)).check(matches(withText("0")));
```
Comment (JoseAlcerreca) [inline app/src/androidTest/java/com/example/android/codelabs/agera/step2/Step2ActivityFinalTest.java:47]: why? IDs are not typed.

File: app/src/main/java/com/example/android/codelabs/agera/step2/Step2Activity.java [hunk 3]
```diff
@@ -31,23 +31,23 @@
...
-    private final MutableRepository<Integer> valueRepository = Repositories.mutableRepository(0);
+    private final MutableRepository<Integer> mValueRepository = Repositories.mutableRepository(0);
 
-    private Repository<String> textValueRepository;
+    private Repository<String> mTextValueRepository;
 
     private Updatable mTextValueUpdatable;
 
-    private TextView mValueTv;
+    private TextView mValueTextView;
```
Comment (JoseAlcerreca) [inline app/src/main/java/com/example/android/codelabs/agera/step2/Step2Activity.java:40]: No strong feelings about this, but ok.

File: build.gradle [hunk 4]
```diff
@@ -5,9 +5,9 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.0-alpha4'
+        classpath 'com.android.tools.build:gradle:2.1.0'
```
Comment (JoseAlcerreca) [inline build.gradle:8]: Please don't change this.

File: app/src/main/java/com/example/android/codelabs/agera/UiUtils.java (hunk 1 above)
Comment (drakeet) [inline app/src/main/java/com/example/android/codelabs/agera/UiUtils.java]: UI is the abbreviation of Interface User, should use all capital.

**Comment by JoseAlcerreca** [comment]
Thanks drakeet for the PR.

However, most of the changes are nits (and some can't be accepted, I've commented on some). I only see a couple of bigger issues like fields not being prefixed with the 'm'. 

Since the codelab would have to be updated with these changes, I'm not going to merge it yet.

File: app/src/androidTest/java/com/example/android/codelabs/agera/step2/Step2ActivityFinalTest.java (hunk 2 above)
Comment (drakeet) [inline app/src/androidTest/java/com/example/android/codelabs/agera/step2/Step2ActivityFinalTest.java:47]: There is no need to use type(such as `tv`, `bt`) in IDs, and in the entire source code of Android SDK is not seen so named. :P

Comment (JoseAlcerreca) [inline app/src/androidTest/java/com/example/android/codelabs/agera/step2/Step2ActivityFinalTest.java:47]: Of course there's no need, but AFAIK there's no rule that says they shouldn't. :)

File: build.gradle (hunk 4 above)
Comment (drakeet) [inline build.gradle:8]: Sorry, but if someone is not using Android Studio 2.2 preview 4, he can not use this version of the gradle, by the way, the latest version of the AS is 2.2 Preview 5 with `2.2.0-alpha5` gradle and cannot use alpha4
image

**Comment by drakeet** [comment]
Thank you for your reply ^ ^ and the reason I pull this request is:

I think, Google's code should be everyone's norms, is a mirror. So when I see minor problem and want to improve it. :P

File: app/src/main/java/com/example/android/codelabs/agera/UiUtils.java (hunk 1 above)
Comment (drakeet) [inline app/src/main/java/com/example/android/codelabs/agera/UiUtils.java]: I checked the document, this point you're right, for readable, it's reasonable and acceptable:

image

https://source.android.com/source/code-style.html

**Pull request closed without merging**